server/monitor/src/main/java/org/apache/accumulo/monitor/servlets/BasicServlet.java

```diff
@@ -264,4 +264,12 @@ protected static void banner(StringBuilder sb, String klass, String text) {
     sb.append("<br />\n<h2 class='").append(klass).append("'>").append(text).append("</h2>\n");
   }
 
+  /**
+   * Creates a {@link Cookie} with the given name and value, also setting the HttpOnly attribute on the cookie.
+   */
+  protected static Cookie createCookie(String name, String value) {
+    Cookie c = new Cookie(name, value);
+    c.setHttpOnly(true);
```

Comment:
The only cookie which actually needed to be protected was the session cookie addressed in ACCUMULO-4676. All of these cookies are simple user UI preferences (table column sort order, etc.) and it doesn't matter if they are manipulated in client-side javascript. Are we certain we're not manipulating them in client-side javascript now? If we are, this will break that.